src/utils/apicalls.js

```diff
@@ -57,4 +57,12 @@ export const addComment = (article_id, username, body) => {
     .catch((err) => {
         console.log(err)
     })
+}
+
+export const deleteComment = (comment_id) => {
+    console.log(comment_id)
+    newsApi
+    .delete(`/api/comments/${comment_id}`).then((response) => {
+        return response
+    })
```

Comment:
how are you handling the errors? What if the user is offline and the comment isn't deleted?